nebula_ros/src/hesai/decoder_wrapper.cpp

```diff
@@ -255,6 +131,12 @@ void HesaiDecoderWrapper::ProcessCloudPacket(
 
   cloud_watchdog_->update();
 
+  auto sys_pc_time_diff = parent_node_.now().seconds() - std::get<1>(pointcloud_ts);
+  // RCLCPP_INFO_STREAM(
```

Comment:
Leave commented code for easy debug later, or delete if no longer needed?

Reply:
Removed in b54891e.